Gaming The Syllabus/Assets/Scripts/Riddles/Riddle.cs

```diff
@@ -0,0 +1,104 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+using System.Xml.Schema;
+using System.Xml;
+using UnityEngine;
+
+public class Riddle : IXmlSerializable
+{
+    public string Name { get; private set; }
+    public string Question { get; private set; }
+
+    public AnswerType TypeOfAnswer { get; private set; }
+
+    private List<RiddleAnswer> _answers = new List<RiddleAnswer>();
+    public RiddleAnswer[] Answers
+    {
+        get => _answers.ToArray();
+    }
+
+    public RiddleAnswer[] CorrectAnswers
+    {
+        get => _answers.FindAll((answer) => answer.IsCorrect).ToArray();
+    }
+
```

Comment:
This is a good reminder to myself to utilize arrow functions more. It definitely keeps things more compact and concise when used appropriately. Great work!